### PR TITLE
Added option for doing software selection read-only

### PR DESCRIPTION
**Pull request by mchf**
update for control.rn?: https://github.com/yast/yast-installation-control/pull/31

the code which uses this option is still WIP

### CODE REVIEW

File: package/skelcd-control-CASP.changes [hunk 1]
```diff
@@ -10,6 +16,7 @@ Tue Dec 13 16:25:14 UTC 2016 - jreidinger@suse.com
 
 - Add keyboard and root password combined dialog to installation
   work-flow (FATE#321764)
+>>>>>>> upstream/master
```
Comment (imobachgs) [inline package/skelcd-control-CASP.changes]: Please, remove this leftover.

File: package/skelcd-control-CASP.changes [hunk 2]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Wed Dec 14 12:43:25 UTC 2016 - mfilka@suse.com
+
+- Added option for doing software selection read-only
```
Comment (imobachgs) [inline package/skelcd-control-CASP.changes:4]: *doing* or *making*? Let's check with our language expert :) @mvidner

**Review by imobachgs — CHANGES_REQUESTED** [review]

File: package/skelcd-control-CASP.changes (hunk 1 above)
Comment (mchf) [reply package/skelcd-control-CASP.changes]: grumpf. thx

File: package/skelcd-control-CASP.changes (hunk 2 above)
Comment (mvidner) [reply package/skelcd-control-CASP.changes:4]: Added *an* option for *making* *the* software selection read-only.

**Comment by mvidner** [comment]
OK, where is the matching code for this option? Still to be written, or existing already? Please put it to the PR description.

**Comment by kobliha** [comment]
@mchf It seems that the PR can't be merged

**Comment by mchf** [comment]
Solved otherwise in the end

**Pull request closed without merging**